Update README.md

nkh0472 opened the pull request:
add the link of mpv-rename to the 'example' section

nkh0472 committed:
Update README.md

add the link of mpv-rename to the 'example' section

CogentRedTester commented:
I'd rather not have 3rd party scripts linked in the main README, as I can't modify them if `mpv-user-input` changes it's API in the future. However, I have just created a wiki page [here](https://github.com/CogentRedTester/mpv-user-input/wiki/Scripts-that-use-mpv%E2%80%90user%E2%80%90input) where you can list the script. It should be publicly editable. This way people can add examples in the future without needing to create PRs.

If you were to change this PR to document the existence of the wiki page in the README, then I would merge it.

nkh0472 committed:
Update README.md

update wiki page link

nkh0472 commented:
> I'd rather not have 3rd party scripts linked in the main README, as I can't modify them if `mpv-user-input` changes it's API in the future. 

That makes sense.

> If you were to change this PR to document the existence of the wiki page in the README, then I would merge it.

Following your suggestion, I have updated the newly created wiki page and added the wiki's link to README:)

CogentRedTester commented:
Looks good, but in the wiki page I'd rather you remove the disclaimer. I think since it's publicly accessible people should be able to infer that they should use some caution.
